test/e2e/validation_test.go

```diff
@@ -468,8 +468,14 @@ func testCSRApproval(t *testing.T) {
 
 	// Scale the Cluster Machine Approver deployment back to 1.
 	expectedPodCount := int32(1)
-	err = testCtx.scaleMachineApproverDeployment(&expectedPodCount)
-	require.NoError(t, err, "failed to scale Cluster Machine Approver pods")
+	err = testCtx.scaleDeployment(machineApproverNamespace, machineApproverDeployment, machineApproverPodSelector,
+		&expectedPodCount)
+	require.NoError(t, err, "failed to scale up Cluster Machine Approver pods")
+
+	// Scale the Cluster Version Operator deployment back to 1.
+	err = testCtx.scaleDeployment(clusterVersionOperatorNamespace, clusterVersionOperatorDeployment,
+		clusterVersionOperatorPodSelector, &expectedPodCount)
+	require.NoError(t, err, "failed to scale up CVO pods")
```

Comment:
I'm not a fan of how we are scaling up and down within different tests. Outside the scope of this PR though.